action/deregister.go

```diff
@@ -27,6 +27,12 @@ func DeregisterCLI(c *cli.Context) error {
 		return errors.New("no VaaS director specified")
 	}
 
+	err := config.GetSecretFromFile(config.VaaSKeyFile)
+	if err == nil {
```

Comment:
This is unusual and confusing. Why not let `GetSecretFromFile` just return error from ioutils?

Reply:
This is actually wrong, it should check if it's NOT nil.